partner-management-default.properties

```diff
@@ -227,7 +234,106 @@ pms.notifications-schedule.fixed-rate=24
+mosip.role.pms.postpartnersapikeyrequestfiltervalues=PARTNER,PARTNER_ADMIN,AUTH_PARTNER,CREDENTIAL_PARTNER,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.postpartnersapikeyrequestsearch=PARTNER,PMS_USER,AUTH_PARTNER,ABIS_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.postpartnersapikeysearch=PARTNER,PMS_USER,AUTH_PARTNER,ABIS_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.putpartnerspolicygroup=PARTNER,PMS_USER,AUTH_PARTNER,CREDENTIAL_PARTNER,PARTNER_ADMIN,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.postpartnerspolicymap=AUTH_PARTNER,ABIS_PARTNER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,ONLINE_VERIFICATION_PARTNER,PARTNER_ADMIN
+mosip.role.pms.patchpartnersgenerateapikey=AUTH_PARTNER,ABIS_PARTNER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.putpartnersapikeypolicies=PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.patchpartners=PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.getpartners=PARTNERMANAGER,PARTNER_ADMIN,partnermanager,ID_AUTHENTICATION,REGISTRATION_PROCESSOR,RESIDENT,CREDENTIAL_ISSUANCE,ID_REPOSITORY
+mosip.role.pms.getpartnersnew=PARTNERMANAGER,PARTNER_ADMIN,partnermanager,ID_AUTHENTICATION,REGISTRATION_PROCESSOR,RESIDENT,CREDENTIAL_ISSUANCE,ID_REPOSITORY
+mosip.role.pms.getpartnersapikey=PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.putpartnerspolicymapping=PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.patchpartnerspolicyapikeystatus=PARTNERMANAGER,PARTNER_ADMIN,AUTH_PARTNER,CREDENTIAL_PARTNER,CREDENTIAL_ISSUANCE,ONLINE_VERIFICATION_PARTNER
+mosip.role.pms.postmisplicense=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.putmisplicense=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.getmisplicense=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.getmisplicensekey=PARTNERMANAGER,PARTNER_ADMIN
+mosip.role.pms.postmispfiltervalues=MISP_PARTNER,MISP,PARTNERMANAGER,PARTNER_ADMIN
+
+
 # query param usd to refer url to redirect after logout
 mosip.iam.post-logout-uri-param-key=post_logout_redirect_uri
 # end session endpoint in OIDC
-mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout
\ No newline at end of file
+mosip.iam.end-session-endpoint-path=/protocol/openid-connect/logout
+# IDP claims mapping file url
+mosip.idp.claims-mapping-file-url= https://raw.githubusercontent.com/mosip/mosip-config/develop-v3/idp-claims-mapping.json
+
+## IDP Service apis for create and update
+pmp-idp.oidc.client.create.rest.uri=https://${mosip.api.public.host}/v1/idp/client-mgmt/oidc-client
+pmp-idp.oidc.client.update.rest.uri=https://${mosip.api.public.host}/v1/idp/client-mgmt/oidc-client
+
+## acr_amr mappings.
+## This is depending on the IDP service configuration.If changes made in IDP service configurations
+## corresponding changes needs to be made here also
+mosip.pms.idp.acr-amr-mappings = {\
```

Comment:
this change seeems to be part of some mapping json. 
Can we get this confirmed again?